app-mobilephone/gammu/gammu-1.36.8.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="http://dl.cihar.com/${PN}/releases/${P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="amd64 x86"
```

Comment:
stabilisation does not work that way, you need to file a stabilisation bug instead.